tests/module_api/test_api.py

```diff
@@ -89,6 +89,19 @@ def test_get_userinfo_by_id__no_user_found(self):
         found_user = self.get_success(self.module_api.get_userinfo_by_id("@alice:test"))
         self.assertIsNone(found_user)
 
+    def test_get_user_ip_and_agents(self):
+        user_id = self.register_user("test_get_user_ip_and_agents_user", "1234")
+        info = self.get_success(self.module_api.get_user_ip_and_agents(user_id))
+        self.assertIdentical(info, [])
+
+    def test_get_user_ip_and_agents__no_user_found(self):
+        info = self.get_success(
+            self.module_api.get_user_ip_and_agents(
+                "@test_get_user_ip_and_agents_user_nonexistent"
+            )
+        )
+        self.assertIdentical(info, [])
```

Comment:
It looks like these only test that the module API returns nothing when it should, could you also add tests to make sure it returns the right data when it should return data please?